<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterDialog</class>
 <widget class="QDialog" name="FilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rx Filter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Filter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="filterEdit">
         <property name="inputMask">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="EXIDEcheckBox">
       <property name="text">
        <string>Extended ID</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="removeAllButton">
       <property name="text">
        <string>Remove All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Set Filters</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Mask</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="maskEdit">
         <property name="inputMask">
          <string/>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>50</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="MIDEcheckBox">
       <property name="text">
        <string>Match EXIDE</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="3">
      <widget class="QCheckBox" name="filterEnabledCheckBox">
       <property name="text">
        <string>Enabled</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Filter Number</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="filterNumberSpinBox">
       <property name="wrapping">
        <bool>true</bool>
       </property>
       <property name="maximum">
        <number>31</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>filterNumberSpinBox</tabstop>
  <tabstop>filterEnabledCheckBox</tabstop>
  <tabstop>maskEdit</tabstop>
  <tabstop>MIDEcheckBox</tabstop>
  <tabstop>filterEdit</tabstop>
  <tabstop>EXIDEcheckBox</tabstop>
  <tabstop>removeAllButton</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>FilterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterNumberSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FilterDialog</receiver>
   <slot>updateDisplay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateDisplay()</slot>
 </slots>
</ui>
